Step-by-step-Tutorials/Running-in-a-Stand-alone-Silo.md

```diff
@@ -30,11 +30,15 @@ static void Main(string[] args)
     Console.WriteLine("Waiting for Orleans Silo to start. Press Enter to proceed...");
     Console.ReadLine();
 
-    Orleans.GrainClient.Initialize("ClientConfiguration.xml");
+    // Orleans comes with a rich XML configuration but we're just going to setup a basic config
+    var config = new Orleans.Runtime.Configuration.ClientConfiguration();
+    config.Gateways.Add(new IPEndPoint(IPAddress.Loopback, 30000));
+
+    GrainClient.Initialize(config);
 }
```

Comment:
hmmm, I see that the silo is using XML config. If you are keeping it that way, be sure to hardcode the gateway port there, such as `ClientConfiguration.LocalhostSilo(30000)`, as the default for programmatic config is now 40000